### DISCUSSION TITLE
Mixed content ERROR

### DISCUSSION THREAD

**Discussion by offerstyle** [wintercms/winter#705] [Q&A]
HI
I installed winter cms on the hosting and when I log in to the admin panel, a (Mixed content) problem is displayed in the console
ScreenCapture
https://disk.yandex.com.am/i/pFkyTgwKn3U-og

hosting connected with cloudfire 
https://account.interbank-aruba.org/

Thanks in advance

**Comment by bennothommo** [comment]
@offerstyle are you using a `http://` address in your `config/app.php` file for the `url`? If so, change that to `https`.

**Reply by offerstyle** [reply]
In config/app.php
Url https://account.interbank-aruba.org/

**Reply by bennothommo** [reply]
Do you have an `.env` file in the project?

**Reply by offerstyle** [reply]
no

**Reply by offerstyle** [reply]
there is a file in the config folder

**Reply by bennothommo** [reply]
You may need to enable [Trusted Proxies](https://wintercms.com/docs/setup/configuration#trusted-proxies) support if you're using a CloudFlare proxy service.

**Reply by offerstyle** [reply]
ok thanks

**Comment by LukeTowers** [comment]
@offerstyle try using the new [Winter.Cloudflare plugin](https://github.com/wintercms/wn-cloudflare-plugin)
